README.md

```diff
@@ -29,22 +31,54 @@ The configuration file `(src/main/resources/config.json)` specifies one or more
 }
 ```
 
-### Usage
+Table names will be of the format [tableNamePrefix]YYYYMMDD for use in table date range queries.
+
+## Job Inserter Configuration
+
+To add a query, you need to modify the configuration file `(src/main/resources/job-inserter-config.json)`
+and add a query under `src/main/resources/queries/`
+
+The configuration file follows the following schema:
+
+```json
+{
+  "projectId": "destination-project",
+  "configuration":
+  {
+    ... 
+  },
+  "every4Hours": [
+    {
+      "destinationTableId": "destination_table_name",
+      "fileName": "queries/query-file-name.sql"
+    }
+  ],
+  "every24Hours": [
+    ...
+  ]
+}
+```
+
+The query result will be stored in `[client-side-events:Aggregate_Tables.destination_table_name]`.
```

Comment:
@ahmedalsudani Isn't the project name configurable in the configuration section rather than being fixed to client-side-events?  

Also, why does the configuration only allow one project / dataset id instead of allowing that configuration in each query?